### DISCUSSION TITLE
Jargon gone mad !

### DISCUSSION THREAD

**Discussion by phpeterAgain** [community/community#60597] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Is there ANY PLAIN ENGLISH explanation of how to use github to run an html file ?  It was 1000x easier to understand Einsteins theories !

**Comment by OshekharO** [comment]
It's super easy

1. Create a GitHub repository

2. Upload your HTML file

3. Enable GitHub Pages: In your repository, go to the "Settings" tab and scroll down to the "GitHub Pages" section. Choose the branch where your HTML file is located (usually the main branch), and click on the "Save" button. GitHub Pages will now publish your HTML file as a website.

4. Access your website: After enabling GitHub Pages, you'll see a message with the URL of your website.

That's it! Your HTML file is now running on GitHub Pages. @phpeterAgain

**Reply by phpeterAgain** [reply]
The point is that the whole process is made impossible UNLESS you learn the JARGON ....... a 'repository' is a folder !!!!!!!!  What if driving instructors invented different words for the brake pedal or the accelerator ..... words that only he knew and like all other imbeciles felt powerful because only he could explain what his JARGON referred to !
